_posts/2025-08-26-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Using Strimzi Metrics Reporter to Expose Kafka Metrics in Prometheus Format"
+date: 2025-10-01
+author: Owen Corrigan
```

Comment:
You need to add yourself in this list here https://github.com/strimzi/strimzi.github.io/blob/main/_data/authors.yml and then referring the entry from here.

Reply:
Done. Thanks for the info